src/cpp/include/UTIL/LCRelationNavigator.h

```diff
@@ -69,24 +70,65 @@ namespace UTIL {
     const EVENT::FloatVec & getRelatedFromWeights(EVENT::LCObject * to) const ;
 
     /** Object with a highest weight that the given from-object is related to.
-     *  LCObject is of type getToType().
+     *  LCObject is of type getToType(). CompareF is a comparison function object
+     *  with the signature bool(float a, float b) which returns ​true if a is less than b.
```

Comment:
```suggestion
     *  LCObject is of type getToType(). CompareF is a comparison function (anything 
     *  callable with the right signature works) that compares the weights of the relations.
     *  The required signature is bool(float a, float b) which returns ​true if a is less than b.
```

Just to drive home the point that it compares the **weights**.